webextension/content-scripts/icon-overlay.js

```diff
@@ -125,9 +125,44 @@ function ytEmbedChecks() {
   }
 }
 
-function ytHomePageHandler(el) {
-  if (el.classList.contains('minvid__overlay__wrapper')) return;
+function registerClick({ el, url }) {
+  return function(ev) {
+    evNoop(ev);
+    browser.runtime.sendMessage({
+      title: 'launch',
+      url: 'https://youtube.com' + url,
+      domain: 'youtube.com',
+      action: getAction(ev)
+    });
+  };
+}
+
+function updateClickIfNeeded(el, { url, props }) {
```

Comment:
Random question, but is it odd that `registerClick()` expects the `el` and `url` to be passed in as a single object/argument, but `updateClickIfNeeded()` separates those into two arguments?

```js
registerClick({ el, url })
// versus
updateClickIfNeeded(el, { url });
```

Reply:
I agree it is a bit weird, this was from the rebase with master. It's a bit of code from a contributor, we can log a new issue, but probably shouldn't be addressed in this pr https://github.com/meandavejustice/min-vid/blob/master/webextension/content-scripts/icon-overlay.js#L140